src/app/work-item/work-item-list/work-item-list-entry/work-item-list-entry.component.html

```diff
@@ -5,6 +5,10 @@
 	></alm-dialog>
 </div>
 <div class="list-group-item" (click)="onSelect($event)" [class.selected]="isSelected()">
+	<!--checkbox to select a WI and move it-->
+	<div class="row-cbh" >
+		<input type="checkbox" [checked]="isChecked()" (click)="toggleEntry($event)" />
```

Comment:
Why even use this function? I see this function is only used here to get the value of chooseWi, so you can remove this function and use **chooseWi** directly instead.